msg/Defect.msg

```diff
@@ -1,10 +1,8 @@
 # DISCLAIMER: this will not be the final messagge to describe a defect.
 # This is a temporary placeholder for demonstration purposes.
 
-# Severity levels
-# 1 = class A
-# 2 = class B
-# 3 = class C
+uint8 number
 uint8 severity
```

Comment:
I see you also removed the enumeration values of the defect type, maybe because it's a repetition of the data contained in the documentation.

Maybe take a look [here](https://github.com/ros2/rosidl/issues/260), but it could be better if we explicitly write here in the message the different constants that are so attached to the message itself.
This can also make programming "safer", especially in C++.

Reply:
ok, I have added them back.